tests/unit/element_test.ts

```diff
@@ -1,40 +1,124 @@
 import { buildFor } from "../../mod.ts";
 import { assertEquals } from "../../deps.ts";
 import { browserList } from "../browser_list.ts";
+const ScreenshotsFolder = "./Screenshots";
+import { existsSync } from "../../src/utility.ts";
 
 for (const browserItem of browserList) {
-  Deno.test("click() | It should allow clicking of elements", async () => {
-    const Sinco = await buildFor(browserItem.name);
-    const page = await Sinco.goTo("https://chromestatus.com");
-    const elem = await page.querySelector('a[href="/roadmap"]');
-    await elem.click();
-    await page.waitForPageChange();
-    await page.assertSee("Roadmap");
-    await Sinco.done();
+  Deno.test("click() | It should allow clicking of elements and update location", async () => {
+    const { browser, page } = await buildFor(browserItem.name);
+    await page.location("https://drash.land");
+    const elem = await page.querySelector(
+      'a[href="https://discord.gg/RFsCSaHRWK"]',
+    );
+    await elem.click({}, true);
```

Comment:
Could we like do this so that the "options" arg is in the end. Reason being, we can make it optional, and so can skip the empty braces. that way this line could become elem.click(true). looks more beautiful right?

Reply:
but then i guess if someone wanted to middle click, they'd now have to do `await click(false, { button: "middle" })`, but i guess that is more pretty

though you can still do `await click(null, true)` (first param doesnt need to be an empty object)

Reply:
yeah but that's the thing, Using the middle click is rarer than right click, which is rarer than default left click. and if using middle click, then there is a good chance that user will pass true to wait for page load

Reply:
So making the default click cleaner could help UX

Reply:
yeah thats fair (though middleclicks wait for new page by default)

Reply:
ive gone with

```ts
function click(options: {
  button?: ...,
  waitFor?: "navigation"
})
```

Makes it simple to use, and felt the waitfor didnt need to be its own param

Reply:
Yes this looks far better